app/js/addListItemArray.js

```diff
@@ -5,7 +5,7 @@ const locateProject = projectName => {
   submitItem();
 };
 
-const appendItem = () => {
+const appendItem = e => {
```

Comment:
Does `appendItem` need all of the event object? Or does it just need `event.target`

Reply:
Neither I was using it to figure out what was going on